dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/task/AbstractTaskTest.java

```diff
@@ -30,7 +30,7 @@ protected static class DummyHandle extends AbstractHandle {}
 
   protected static Handle HANDLE = new DummyHandle();
 
-  protected static class MemoryByteSink extends ByteSink {
+  public static class MemoryByteSink extends ByteSink {
```

Comment:
An alternative to this is to let RedshiftClusterNodesTaskTest extend AbstractTaskTest . WDYT?

Reply:
good idea, other test classes already extend AbstractTaskTest so this solution should be fine.

Reply:
done, inner class has to stay public because RedshiftClusterNodesTaskTest is outside of task package, but this allows to get rid off MemoryByteSink import.